pcsx2-qt/Settings/EmulationSettingsWidget.cpp

```diff
@@ -60,7 +60,10 @@ EmulationSettingsWidget::EmulationSettingsWidget(SettingsDialog* dialog, QWidget
 		tr("Sets the fast forward speed. This speed will be used when the fast forward hotkey is pressed/toggled."));
 
 	dialog->registerWidgetHelp(m_ui.slowMotionSpeed, tr("Slow Motion Speed"), tr("User Preference"),
-		tr("Sets the slow motion speed. This speed will be used when the slow motion hotkey is pressed/toggled."));
+		tr("Sets the slow motion speed. This speed will be used when the slow motion hotkey is pressed/toggled."));	
+
+	dialog->registerWidgetHelp(m_ui.speedLimiter, tr("speed Limiter"), tr("checked"),
+		tr("Limits emulation speed"));
```

Comment:
Period, also better to expand it to say something like "Limits the emulation to the appropriate framerate for the currently running game."